### PR TITLE
Fix debug on modals

### LINKED ISSUE #5916: sidebar is meshing up debug output on modals
#### Modals rendering is weird when debug is on

On the latest staging try:
create a user note
Enable debug
go to administrator/index.php?option=com_users&view=users
click on the display  note button
observe the rendered modal
Broken:
screen shot 2015-01-29 at 4 07 23

If we take out the j-toggle-main, the j-toggle-main and set width to 97% in the respected divs everything is back to normal:
screen shot 2015-01-29 at 4 07 03

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Testing

 On the latest staging try:
create a user note
Enable debug
go to administrator/index.php?option=com_users&view=users
click on the display note button
observe the rendered modal
Apply patch and reload the page and the modal
#### Preview

Before:
screen shot 2015-01-29 at 4 07 23

After:
screen shot 2015-01-29 at 4 40 20

### CODE REVIEW

**Comment by waader** [comment]
@test works!

**Comment by dgt41** [comment]
@JoomliC Can you review this one, as I am modifying you code here 😉

**Comment by losedk** [comment]
@test works!

File: administrator/templates/isis/js/template.js [hunk 1]
```diff
@@ -138,13 +144,13 @@
 				$('#j-toggle-sidebar-icon').removeClass(closed_icon).addClass(open_icon);
 				$('#j-toggle-sidebar-button').attr('data-original-title', Joomla.JText._('JTOGGLE_HIDE_SIDEBAR'));
 
-				if (body_width > 768 && main_height < sidebar_height)
+				if (!isComponent && body_width > 768 && main_height < sidebar_height)
 				{
-					$('#system-debug').css('width', this_main+'%');
+						$('#system-debug').css('width', this_main+'%');
```
Comment (zero-24) [inline administrator/templates/isis/js/template.js]: @dgt41 any reason to add a double tab here?

File: administrator/templates/isis/js/template.js [hunk 2]
```diff
@@ -138,13 +144,13 @@
...
 				}
-				else
+				else if (!isComponent)
 				{
-					$('#system-debug').css('width', this_content+'%');
+						$('#system-debug').css('width', this_content+'%');
```
Comment (zero-24) [inline administrator/templates/isis/js/template.js]: same here.

File: administrator/templates/isis/js/template.js (hunk 1 above)
Comment (dgt41) [inline administrator/templates/isis/js/template.js]: oops thats from the last submit. correction coming up

Comment (zero-24) [inline administrator/templates/isis/js/template.js]: :+1:

**Comment by brianteeman** [comment]
two good tests moving to RTO<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5917">issues.joomla.org/joomla-cms/5917</a>.</sub>

**Comment by JoomliC** [comment]
<blockquote>
@JoomliC Can you review this one, as I am modifying your code here :wink:
</blockquote>
Sorry for slow response...
Note that it's not "my" code, but "Joomla contributors" code (a work of many happy volunteers!) ;-)

@test success : confirm the RTC :+1:

**Pull request merged**